src/services/assetService.test.ts

```diff
@@ -150,6 +151,7 @@ describe("Asset Service", () => {
                     state: AssetState.Tagged,
                 },
                 regions: [],
+                version: "",
             };
```

Comment:
I would expect this test to fail right now - If we are setting version to empty string the `assetService.save(...)` should be applying the version from the package.json therefore making the equality comparison fail below.

Reply:
Not really. The assetService.save() method just save an asset metadata it receives in input.  In that specific test the goal was just to check this method that don't really care about the value of asset metadata. The thing that that property is not optional just enforce that at least it as some value and an empty string is still a value.   Btw real project version value is enforced in the redux.  All the test that use the redux save project has been updated in this PR asserting for the valid project version.